web_src/less/_base.less

```diff
@@ -1526,12 +1538,12 @@ a.ui.label:hover {
 }
 
 .lines-num {
+  &:extend(.unselectable);
```

Comment:
`user-select: none;` will be more readable to someone unitiated to Less. Vendor prefixes are not needed.